packages/core/src/test/shared/filesystemUtilities.test.ts

```diff
@@ -40,29 +39,25 @@ describe('filesystemUtilities', function () {
     })
 
     describe('getNonexistentFilename()', function () {
-        const methods = [getNonexistentFilename, getNonexistentFilenameSync]
-
-        methods.forEach(methodUnderTest => {
-            it(`${methodUnderTest.name}: failure modes`, async function () {
-                await assert.rejects(async () => methodUnderTest('/bogus/directory/', 'foo', '.txt', 99))
-                await assert.rejects(async () => methodUnderTest('zzz', 'foo', '.txt', 99))
-            })
-            it(`${methodUnderTest.name}: returns a filename that does not exist in the directory`, async function () {
-                const dir = tempFolder
-                await writeFile(path.join(dir, 'foo.txt'), '', 'utf8')
-                await writeFile(path.join(dir, 'foo-0.txt'), '', 'utf8')
-                await writeFile(path.join(dir, 'foo-1.txt'), '', 'utf8')
-                await writeFile(path.join(dir, 'foo-2.txt'), '', 'utf8')
-                assert.strictEqual(await methodUnderTest(dir, 'foo', '.txt', 99), 'foo-3.txt')
-                assert.strictEqual(await methodUnderTest(dir, 'foo', '', 99), 'foo')
-            })
-            it(`${methodUnderTest.name}: returns "foo-RANDOM.txt" if max is reached`, async function () {
-                const dir = tempFolder
-                await writeFile(path.join(dir, 'foo.txt'), '', 'utf8')
-                await writeFile(path.join(dir, 'foo-1.txt'), '', 'utf8')
-                // Looks like "foo-75446d5d.txt".
-                assert.ok(/^foo-[a-fA-F0-9]{8}.txt$/.test(await methodUnderTest(dir, 'foo', '.txt', 1)))
-            })
+        it(`getNonexistentFilename: failure modes`, async function () {
```

Comment:
since you removed getNonexistentFilenameSync should we remove 'getNonexistentFilename' at the start of the test names since it's already in the describe?